flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamContextEnvironment.java

```diff
@@ -38,14 +38,10 @@
 	private final ContextEnvironment ctx;
 
 	protected StreamContextEnvironment(ContextEnvironment ctx) {
+		super(GlobalConfiguration.loadConfiguration().getInteger(
```

Comment:
This is bound to cause problems, as it relies on that fact that where ever this is called the proper environment variables are set.

I think we need to get rid of all ad-hoc loading of configuration and not introduce more places.